### PR TITLE
French mayors

### PR DESCRIPTION
**Pull request by dhdaines**
Add mayors of French municipalities from the RNE

Closes https://github.com/opensanctions/crawler-planning/issues/85

### CODE REVIEW

File: datasets/fr/maires/crawler.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    )
+    context.log.debug(f"Unique ID {person.id}")
+    h.apply_name(person, given_name=given_name, last_name=family_name, lang="fra")
+    if birth_date:
+        person.add("birthDate", h.parse_date(birth_date, DATE_FORMATS))
+    position = h.make_position(
+        context,
+        name=f"Mayor of {municipality}",
+        country="fr",
+        subnational_area=subnational_area,
```
Comment (jbothma) [inline datasets/fr/maires/crawler.py:56]: if their jurisdiction is the municipality, that is what we'd like here - the idea is that subnational_area narrows the jurisdiction from the country_code to this area for subnational

File: datasets/fr/maires/crawler.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+import csv
+from typing import Dict
+
+from zavod import Context
+from zavod import helpers as h
+from zavod.logic.pep import categorise
+
+UNUSED_FIELDS = [
+    "Code sexe",
```
Comment (jbothma) [inline datasets/fr/maires/crawler.py:13]: sex or gender is also nice to have, acknowledging that we're probably not treating it with as much nuance as we should, but the aim is just to reflect the source

**Review by jbothma — COMMENTED** [review]
Superbe!

Merci!

I'll comment two tiny things but I'll make the changes and merge while I have a copy open.

File: datasets/fr/maires/crawler.py (hunk 2 above)
Comment (dhdaines) [reply datasets/fr/maires/crawler.py:13]: Ah, of course! Yes also useful for disambiguating, I just didn't bother to look in the data dictionary to see what the right attribute name was.

**Pull request merged**